Shubham sharma (starkadam999@gmail.com)

Shubham-aidev opened the pull request:
**my code for moduel-1, week 1 to 2 tasks with 
1 accept aadhar as pdf , jpg , png , jepg 
2 working ocr script to extract data from two types of aadhar layouts
3 extract data from raw text
4 standardize and cleaning of data
5 cleaned data csv as output**

Shubham-aidev committed:
added full pipeline main.py , ocr_extractor.py , field_extractor.py , cleaning_utils.py ,raw_docs

Shubham-aidev committed:
Create readme1.md

Shubham-aidev committed:
24th august  readme1.md

springboard1233 commented:
Dear Shubham,
I have seen your code , and it seems good and well done as of now only you
have started working on 1st milestone, dropping some key features for you
as you are working alone so just have a look as to whether all frontend and
backend part are as per required details or not.

In *Milestone 1*, the goal is to *extract Aadhaar details using OCR* and
build a *basic UI + API setup* where you can:

   -

   Upload an Aadhaar card (image/PDF)
   -

   Extract Aadhaar details using OCR
   -

   Display extracted details on the screen
   -

   Send extracted data to backend for verification & storage

   # Tasks for Backend POV-
   *Task* *Description* *Tools/Framework*
   *API Development* Create Flask/Django API endpoints Python + Flask/Django
   *File Handling* Accept Aadhaar image/PDF Flask request.files
   *OCR Extraction* Use Tesseract/OpenCV to extract data pytesseract
   *Data Cleaning* Apply regex to extract Name, DOB, Aadhaar No re module
   *Database Setup* Save verified data MySQL / MongoDB
   *Error Handling* Handle invalid Aadhaar uploads Flask exception handling

   FINAL CLARIFICATION :-

   *Frontend :*
      -

      Create *UI* for Aadhaar upload & results display.
      -

      Integrate *API calls* with backend.
      -

      Make the interface *clean, user-friendly, and responsive*.
      *Backend :*
      -

      Build *REST APIs* for Aadhaar OCR extraction.
      -

      Handle *file uploads*, *OCR processing*, *regex-based parsing*.
      -

      Save data into a database and return responses to frontend.



Thanks $ Regards
Kapil Pawar
Mentor~ AI-Powered Identity Verification and Fraud Detection for KYC
Compliance.

> Message ID: ***@***.***>
>

Shubham-aidev committed:
add endpoints api for login, signup , extract api, save api , fetch api and sample frontend templates home.html login.html and signup.html , add .gitignore

Shubham-aidev committed:
Create readme_for _frontent_people.md

this is shubham sharma contribution to team -A backend part .
on wednesday 27 august for project

Shubham-aidev committed:
readme_week2.md (on 27 august )

shubham sharma contribution for team A in backend part

Shubham-aidev committed:
update week-2 according to addons

Shubham-aidev committed:
optimised for AI models

Shubham-aidev committed:
MILESTONE -3 COMPLETED BY SHUBHAM SHARMA (starkadam999@gmail.com)
